js/uid2-sdk-3.0.0.ts

```diff
@@ -237,108 +238,103 @@ export class UID2 {
             status: status,
             statusText: statusText
         };
-        this._opts.callback(result);
-
-        if (advertisingToken) {
-            promises.forEach(p => p.resolve(advertisingToken));
-        } else {
-            promises.forEach(p => p.reject(new Error(statusText)));
-        }
+        if (this._opts?.callback) this._opts.callback(result);
```

Comment:
what do we use this `opts.callback` for? same as push a callback and listen to `EventType.IdentityUpdated`?

Reply:
Legacy support - this is the current way to integrate with the SDK (2.0 and earlier), but it's not flexible enough so we need the new array-style callbacks.

Reply:
Understand now, thanks.